recipes/pylatexenc/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015-2019 Philippe Faist
```

Comment:
Looks like the license file is in the Github tarball, so please remove the file from here.